### DISCUSSION TITLE
yabai3: i3 sanity wrapper around yabai

### DISCUSSION THREAD

**Discussion by sebastianst** [asmvik/yabai#1973] [Show and tell]
[yabai3](https://github.com/sebastianst/yabai3) is a simple zsh script that provides a few commands which imitate the way i3 behaves under the same action, but uses `yabai` to accomplish the intended effects.

Among a few others, it provides `focus`, `move`, and `resize` commands that imitate the same commands of i3. In particular, they are stack-aware.

Feedback and PRs welcome!

**Comment by qhsong** [comment]
Nice Work~
